server/build/webpack.js

```diff
@@ -33,12 +34,28 @@ export default async function createCompiler (dir, { hotReload = false, dev = fa
 
   const nodeModulesDir = join(__dirname, '..', '..', '..', 'node_modules')
 
+  const alias = {
+    'babel-runtime': babelRuntimePath
+  }
+
+  // Alias next's API in the next dev mode.
+  // This is only required when developing Next.js
+  if (process.env.NEXT_DEV) {
```

Comment:
Earlier because of these `require.resolve` calls, we had to ship Next modules inside all the pages, even they shipped inside the bundle.

This fix removes that. When we are developing(and testing) we need to enable this flag.